draft-iab-privacy-partitioning.md

```diff
@@ -227,6 +227,34 @@ from observing both identities and correlating them. As such, preventing correla
 requires separating contexts, such as by using proxying to conceal a client IP address
 that would otherwise be used as an identifier.
 
+## Approaches to Partitioning
+
+While all of the partitioning protocols described in this document use the
+basic techniques of encryption and connection separation, each one has a
```

Comment:
```suggestion
While all of the partitioning protocols described in this document separate contexts
using cryptography, connection separation, or a combination of the two, each one has a
```

Because DAP doesn't need connection separation -- it just uses secret sharing.

Reply:
```suggestion
While all of the partitioning protocols described in this document create
separate contexts using encryption and/or connection separation, each one has a
```